tsdb_k8s_queries/operations/default.json

```diff
@@ -0,0 +1,1085 @@
+{% set end_time = "2023-05-16T21:00:00.000Z" %}
+{% set time_intervals = {"15_minutes": ["30s", "2023-05-16T20:45:00.000Z"], "2_hours": ["1m", "2023-05-16T19:00:00.000Z"], "24_hours": ["30m", "2023-05-15T21:00:00.000Z"]} %}
```

Comment:
The above two seem to be used also in `challenges/default.json`; does it make more sense to move them, as well as the variables from `challenges/default.json`:

```
{% set search_iterations = 20 %} 
{% set search_warmup_iterations = 50 %}
{% set target_interval = 4 %}
```

to a centralized place, e.g. at the top of `track.json` (right after `{% import "rally.helpers" as rally with context %}`) for better discoverability? If you choose to do that, remember to update the references in `README.md`.

Reply:
This makes sense to me. I will update this PR.

Reply:
> This makes sense to me. I will update this PR.

LGTM also after https://github.com/elastic/rally-tracks/pull/373/commits/5ea2206e02119b7f2f02a7fc803634082a99aec6